README.md

```diff
@@ -1,2 +1,42 @@
-# cloud-platform-prometheus
+# Cloud-Platform Prometheus
+==================
+
+This repository will allow you to create a monitoring namespace in a MoJ Cloud Platform cluster. It will also contain the neseccary values to perform an installation of Prometheus-Operator and Kube-Prometheus. 
+
+  - [Pre-reqs](#prereq)
+  - [Creating a monitoring namespace](#installation)
+  - [Installing Prometheus-Operator](#output-example)
+  - [Installng Kube-Prometheus](#usage)
+  - [Exposing the port](#features-and-advantages-of-this-project)  
+  - [How to tear it all down](#am-i-missing-some-essential-feature)
+  - [Contributing](#contributing)
```

Comment:
Looks like a good set of steps but I'd also say -- this is for someone else on the team right now so you can probably rely on the code a bit and just point people to the things they need to add/change/manage.

Reply:
Really good point. Will be sure to take that into consideration going forward.